### ISSUE TITLE
HTML errors

### ISSUE THREAD

**Issue by jfoclpf** [jfoclpf/autocosts#71]
Solve these [issues](https://validator.w3.org/nu/?doc=http%3A%2F%2Fautocosts.work%2FUS).

Particularly this one:
`Element h5 not allowed as child of element button in this context. (Suppressing further errors from this subtree.)`

which is the most common and coming from `results.hbs`, [here](https://github.com/jfoclpf/autocosts/blob/newUI/src/views/partials/results.hbs#L93).

```
            <button class="accordion">
                <h5 class="car-finance_color">{{{credit}}}</h5>
                <span class="periodic_costs_credit"></span>
            </button>
```

Replacing `button` to `div` solves this issues. Do you agree @mcristaldo ?

**Comment by mcristaldo** [comment]
Solved, please check it out.

Thank you

**Comment by jfoclpf** [comment]
Thank you

Solved with cc741d66371ae1e75d7784b95d1a65be9a599d67, 2dce9a74e79b12ac356d54ec8a2060a813c960e5, 2799b95be7207f9467e0840322b27e35273de141, 9f32e3ed03aff5102f2a02fbd1c89225f9c276d5 and b482d4659476d1512e2afdc003c2350d05fea2f0
